src/Equinor.ProCoSys.IPO.Query/GetInvitationsQueries/GetInvitations/GetInvitationsQueryHandler.cs

```diff
@@ -31,20 +32,58 @@ public async Task<Result<InvitationsResult>> Handle(GetInvitationsQuery request,
             // count before adding sorting/paging
             var maxAvailable = await queryable.CountAsync(token);
 
-            var enumerable = AddSorting(request.Sorting, queryable.AsEnumerable());
-            enumerable = AddPaging(request.Paging, enumerable);
+            queryable = AddSorting(request.Sorting, queryable);
```

Comment:
Change the name of the variable "queryable" to what it actually is for better readability. Do not use variable names like enumerable, queryable.